src/pages/Facility/services/inventory/externalSupply/requestOrder/RequestOrderShow.tsx

```
@@ -439,6 +440,46 @@ export function RequestOrderShow({
                 </div>
               </div>
 
+              <div>
+                <label className="text-sm font-medium text-gray-700">
+                  {t("tags_other")}
+                </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)